jtsync/jtsync.rb

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env ruby
+#
+# Copyright 2016, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ENV["CURRENT_GEMFILE"] ||= File.expand_path("Gemfile", __FILE__)
+
+require "bundler"
+Bundler.setup(:default)
+require "trello"
+
+
+class Failed < StandardError
+end
+
+
+def board
+  @trello_board ||= if ENV.has_key? "TRELLO_BOARD"
```

Comment:
I'm pretty sure HoundCI would hate that if we had it set up on this repo ;-)  Split the `if` onto the next line?